src/libs/Navigation/AppNavigator/AuthScreens.tsx

```diff
@@ -193,9 +198,9 @@ class AuthScreens extends React.Component {
                 });
             },
             searchShortcutConfig.descriptionKey,
-            searchShortcutConfig.modifiers,
+            [...searchShortcutConfig.modifiers],
             true,
-        );
+        ) as UnsubscribeSearchShortcut;
```

Comment:
shouldn't this be done by KeyboardShortcut module? so we dont need to us `as` here?